cypress/e2e/API Tests/getCustomers.spec.js

```diff
@@ -0,0 +1,11 @@
+describe('API Test - Return customers', () => {
+  it('Should return a list of customers', () => {
+    cy.request('GET', 'http://localhost:3001/customers')
+      .then((response) => {
+        expect(response.status).to.eq(200);
+        expect(response.body).to.have.property('customers');
+        expect(response.body.customers).to.be.an('array');
+        expect(response.body.customers.length).to.be.greaterThan(0);
+      });
+  });
+});
```

Comment:
```suggestion
});

```

É recomendado deixar uma linha em branco ao final de cada arquivo.

Dessa forma, ferramentas como o GitHub não exibem o ícone de `No newline at end of file`.

Isso é uma boa prática e convenção que deve ser seguida.